pubspec.yaml

```diff
@@ -2,10 +2,10 @@ name: font_awesome_flutter
 description: The Font Awesome Icon pack available as Flutter Icons. Provides 1500 additional icons to use in your apps.
 author: Brian Egan <brian@brianegan.com>
 homepage: https://github.com/brianegan/font_awesome_flutter
-version: 8.5.0
+version: 8.6.0
 
 environment:
-  sdk: ">=2.0.0-dev.28.0 <3.0.0"
+  sdk: ">=2.2.2 <3.0.0"
```

Comment:
Was this necessary for one of the changes? I don't think it'd be much of a problem since Flutter often runs a newer version of Dart, but I always prefer supporting the widest possible range.

This would require us to ship a `9.x` since it'd be a breaking change for some folks, only reason I ask.

Reply:
I understand. I updated the SDK because I wanted to use the collection-loops (`[a, b, for (...) c, ]`) but it's purely syntax sugar so I can rewrite it to work with the old SDK constraints.

Reply:
Cool! Don't think it should be a problem / require a change, since I imagine most folks will be on a newer version.